backend/.editorconfig

```
@@ -0,0 +1,378 @@
+root = true
+
+# All files
+[*]
+indent_style = space
+
+# Xml files
+[*.xml]
+indent_size = 2
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+tab_width = 4
+
+# New line preferences
+insert_final_newline = false
```

Is always better in order to avoid the "Missing line" symbol on GitHub.
```suggestion
insert_final_newline = true
```